src/test/java/reposense/commits/CommitInfoExtractorTest.java

```diff
@@ -1,20 +1,23 @@
 package reposense.commits;
 
 import java.util.Calendar;
+import java.util.Collections;
 import java.util.Date;
 import java.util.List;
 
 import org.junit.Assert;
 import org.junit.Test;
 
 import reposense.commits.model.CommitInfo;
+import reposense.model.Author;
 import reposense.template.GitTestTemplate;
 import reposense.util.TestUtil;
 
 public class CommitInfoExtractorTest extends GitTestTemplate {
 
     @Test
     public void withContentTest() {
+        config.setAuthorList(Collections.singletonList(new Author(".*")));
```

Comment:
Seems like you already have `CommandRunnerTest#getNoFilterAuthor()` to create `new Author(.*)`. 

Why not reuse it for clarity?